src/util/postgres_client.js

```diff
@@ -1284,6 +1310,33 @@ class PostgresClient extends EventEmitter {
             if (pg_client) await pg_client.end();
         }
     }
+
+    async upgradeSequence(name) {
+        let pg_client;
+        try {
+            pg_client = new Client({ ...this.new_pool_params, database: undefined });
+            await pg_client.connect();
+            const res = await pg_client.query(`SELECT count(*) FROM pg_tables WHERE tablename  = '${name}';`);
+            const count = Number(res.rows[0].count);
+            if (count === 0) {
+                dbg.log0(`⛔️ Table ${name} not found`);
+                return;
+            }
+            dbg.log0(`✅ Table ${name} is found`);
+            const mongoSeq = new MongoSequence({ name, client: this });
```

Comment:
why `MongoSequence`?

Reply:
To calculate the [start value](https://github.com/baum/noobaa-core/blob/1e7f1aab2c3bbefc45fccb129b21eb8306bbb8bd/src/util/postgres_client.js#L1327) for the native postgres [sequence creation](https://github.com/baum/noobaa-core/blob/1e7f1aab2c3bbefc45fccb129b21eb8306bbb8bd/src/util/postgres_client.js#L1330).

Reply:
but why mongo? we are upgrading from postgres table

Reply:
during the upgrade, the sequence is still in so to speak "mongo" format,  i.e. a single doc in this collection.... [`MongoSequence.nextsequence()`](https://github.com/baum/noobaa-core/blob/1e7f1aab2c3bbefc45fccb129b21eb8306bbb8bd/src/util/mongo_client.js#L85-L92) contains the logic which knows to get the (next) value if the sequence